### DISCUSSION TITLE
2FA Authentication issues

### DISCUSSION THREAD

**Discussion by Margaret6556** [community/community#17437] [Other Feature Feedback, Questions, & Ideas]
Help.  I'm new to Github.  I am not able to set up my 2FA because it won't continue after I enter the code. Can someone please assist? Thank you.

**Comment by queenofcorgis** [comment]
Hi there!

We're glad you're here. This space was created to give feedback on GitHub. For support and questions, I recommend using our [Community Forum](https://github.community/) or for account-specific inquiries, reaching out to our [Support team ](https://support.github.com/request)as they'll be better able to assist :octocat: 😄 

Cheers!

**Comment by Zi-Tao** [comment]
@Margaret6556 https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication
